src/main.js

```diff
@@ -525,6 +527,49 @@ class MiniGraphCard extends LitElement {
       </svg>`;
   }
 
+  renderGridLines() {
+    const {
+      height, hours_to_show, show,
+    } = this.config;
+    const grid_line_type = show.grid_line_type ? show.grid_line_type : false;
+    const grid_line_frequency = show.grid_line_frequency ? show.grid_line_frequency : 2;
+
+    const containerWidth = 500;
+    let numLines;
+
+    let rounded_hours_to_show = Math.round(hours_to_show);
+    if (rounded_hours_to_show < 1) rounded_hours_to_show = 1;
+
+    switch (grid_line_type) {
+      case '5minute':
+        numLines = rounded_hours_to_show * 12;
+        break;
+      case 'hour':
+      default:
+        numLines = rounded_hours_to_show;
+        break;
+      case 'day':
+        numLines = Math.round(rounded_hours_to_show / 24);
+        break;
+      case 'week':
+        numLines = Math.round(rounded_hours_to_show / 168);
+        break;
+    }
+    const xRatio = containerWidth / numLines;
+    const lines = [];
+
+    for (let i = 0; i < numLines; i += 1) {
+      const x = xRatio * (i + 0.5);
+      if (i % grid_line_frequency > 0) {
+        lines.push(svg`<line x1=${x} y1="0" x2=${x} y2=${height} stroke="var(--divider-color)" stroke-width="0.5"/>`);
+      } else {
+        lines.push(svg`<line x1=${x} y1="0" x2=${x} y2=${height} stroke="rgb(from var(--divider-color) R G B /0.5)" stroke-width="0.5"/>`);
```

Comment:
Let's make it more complex - but also more flexible:
```
stroke=var(--mini-graph-card-large-grid-color, rgb(from var(--divider-color) R G B /0.5))
```

Reply:
I don't see the added value here

Reply:
If CSS theme vars are used - users may customize a look dependently on their needs or their accessibility (a11y).
Customizing vars is possible either by custom themes or card-mod.

Reply:
Good to know